fda-model/metrics/ConsequenceDistributionResults.cs

```diff
@@ -286,16 +310,17 @@ public double ConsequenceExceededWithProbabilityQ(double exceedanceProbability,
         /// <returns></returns>
         public ConsequenceDistributionResult GetConsequenceResult(string damageCategory, string assetCategory, int impactAreaID)
         {
-            foreach (ConsequenceDistributionResult damageResult in _consequenceResultList)
+            //foreach (ConsequenceDistributionResult damageResult in _consequenceResultList)
+            for (int i = 0; i < _consequenceResultList.Count; i++)
             {
                 //The impact area should always be equal because a consequence result reflects 1 impact area and a consequence resultS reflects 1 impact area   
-                if (damageResult.RegionID.Equals(impactAreaID))
+                if (_consequenceResultList[i].RegionID.Equals(impactAreaID))
```

Comment:
In general, if we've validated something upstream (like only using one impact area) then I think it's really better to not check for it downstream again. It's likely harmless, but it adds the illusion that there is some possibility that equals might fail, and makes it harder to understand how the code works. 

In short, it just gets confusing. You've got a comment here, which satisfys any concerns, but just wanted to bring that up incase it comes up again.

Reply:
I will refactor.